docs/docs/reference/05-oauth-providers/ping-id.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
As per https://github.com/nextauthjs/next-auth/pull/6614/files#r1096673092, this file can be deleted, as we would generate the API reference from the provided JSDoc comments.

Reply:
The file has been deleted